cli/src/main/java/com/devonfw/tools/ide/process/ProcessContextImpl.java

```diff
@@ -165,6 +165,8 @@ public ProcessResult run(ProcessMode processMode) {
         this.processBuilder.redirectOutput(Redirect.PIPE).redirectError(Redirect.PIPE);
       } else if (processMode.isBackground()) {
         modifyArgumentsOnBackgroundProcess(processMode);
+      } else {
+        this.processBuilder.redirectInput(Redirect.INHERIT);
```

Comment:
ah, we only had redirect output and error but never input.
Great finding, awesome 👍 
BTW: We could create a cleanup PR to add 3 methods to `ProcessMode` returning `Redirect`:
* getRedirectOutput
* getRedirectError
* getRedirectInput

If needed `ProcessMode.BACKGROUND*` could return `null` for some of these methods and if `null`, then the `redirect` is not configured in `processBuilder`.

Then we can make this code much more clean and understandable.
However, you fix the actual problem here functionally correct.
So lets merge this as is and create a new PR for the cleanup.

Reply:
Created issue #1306